api_tests/users/views/test_user_nodes_list.py

```diff
@@ -245,10 +245,66 @@ class TestNodeListFiltering(NodesListFilteringMixin):
     def url(self):
         return '/{}users/me/nodes/?'.format(API_BASE)
 
-
 @pytest.mark.django_db
 class TestNodeListDateFiltering(NodesListDateFilteringMixin):
 
     @pytest.fixture()
     def url(self):
         return '/{}users/me/nodes/?'.format(API_BASE)
+
+@pytest.mark.django_db
+class TestNodeListPermissionFiltering:
+
+    @pytest.fixture()
+    def creator(self):
+        return AuthUserFactory()
```

Comment:
Tiny nit: `creator.auth` isn't used anywhere, so you can make the test slightly faster by using `UserFactory`.